### PR TITLE
worked on the grade_wise_salary_master(showing the listing, edit and …

### PR DESCRIPTION
**Pull request by riyathakur11**
worked on the grade_wise_salary_master functionality

### CODE REVIEW

File: app/Http/Controllers/GradeSalaryMasterController.php [hunk 1]
```diff
@@ -50,8 +50,66 @@ public function store(Request $req)
...
+    }
+
+    public  function allBasicGradeSalary()
+    {
+        $allbasicgradesal = GradeWiseSalaryMaster::with('grade')->get()->toArray();
+        //dump($allbasicgradesal); dd();
+        return view('GradeSalaryMaster.allbasicgradesalarymaster', compact("allbasicgradesal"));
+    }
+
+    public function edit_basic_salary($id)
```
Comment (kumarisavita) [inline app/Http/Controllers/GradeSalaryMasterController.php:63]: Please do some commenting over functions as it's discussed earlier.

File: resources/views/GradeSalaryMaster/allbasicgradesalarymaster.blade.php [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+      <th scope="col">#</th>
+      <th scope="col">Head Title</th>
+      <th scope="col">Formula</th>
+      <th scope="col">Amount</th>
+      <th scope="col">Grade</th>
+      <th scope="col">Action</th>
+    </tr>
+  </thead>
+  <tbody>
+  @if(session()->has('message'))
```
Comment (kumarisavita) [inline resources/views/GradeSalaryMaster/allbasicgradesalarymaster.blade.php:22]: Please improve code indentation here

File: resources/views/GradeSalaryMaster/allbasicgradesalarymaster.blade.php [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+      <td>{{ $val['amount'] }}</td>
+      <td>{{ $val['grade']['grade'] }}</td>
+      <td>
+        <a href="{{ url('edit_basic_salary/' . $val['id']) }}" class="btn btn-primary"><i class="bi bi-pencil"></i></a>
+        <a href="#" class="btn btn-danger delete-btn" data-id="{{ $val['id'] }}"><i class="bi bi-trash"></i></a>
+
+      </td>
+    </tr>
+    @endforeach
+</tbody>
```
Comment (kumarisavita) [inline resources/views/GradeSalaryMaster/allbasicgradesalarymaster.blade.php:42]: code indentation here as well.

File: resources/views/GradeSalaryMaster/allbasicgradesalarymaster.blade.php [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+           </div>
+         </div>
+       </div>
+     </div>
+</form>
+
+@endsection
+@section('js_scripts')
+<script>
+ $(document).ready(function() {
```
Comment (kumarisavita) [inline resources/views/GradeSalaryMaster/allbasicgradesalarymaster.blade.php:70]: why we have put jQuery settimeout method here, can you please share with me.

**Review by kumarisavita — CHANGES_REQUESTED** [review]

File: resources/views/GradeSalaryMaster/allbasicgradesalarymaster.blade.php (hunk 4 above)
Comment (riyathakur11) [reply resources/views/GradeSalaryMaster/allbasicgradesalarymaster.blade.php:70]: To hide the flash message

File: app/Http/Controllers/GradeSalaryMasterController.php (hunk 1 above)
Comment (riyathakur11) [reply app/Http/Controllers/GradeSalaryMasterController.php:64]: Done

File: resources/views/GradeSalaryMaster/allbasicgradesalarymaster.blade.php (hunk 3 above)
Comment (riyathakur11) [reply resources/views/GradeSalaryMaster/allbasicgradesalarymaster.blade.php]: Done

File: resources/views/GradeSalaryMaster/allbasicgradesalarymaster.blade.php (hunk 2 above)
Comment (riyathakur11) [reply resources/views/GradeSalaryMaster/allbasicgradesalarymaster.blade.php]: Done

**Review by kumarisavita — APPROVED** [review]

**Pull request merged**